### PR TITLE
Proxy protocol frame log

### PR DESCRIPTION
**Pull request by JimMoen**

### CODE REVIEW

File: src/emqx_connection.erl [hunk 1]
```diff
@@ -589,6 +589,11 @@ parse_incoming(Data, Packets, State = #state{parse_state = ParseState}) ->
             NState = State#state{parse_state = NParseState},
             parse_incoming(Rest, [Packet|Packets], NState)
     catch
+        error:proxy_protocol_config_disabled:_Stk ->
```
Comment (zmstone) [inline src/emqx_connection.erl:592]: Do not match stacktrace if not needed.
It’s costs

**Review by zmstone — CHANGES_REQUESTED** [review]

File: src/emqx_connection.erl (hunk 1 above)
Comment (JimMoen) [reply src/emqx_connection.erl:592]: Ignored var still costs?

Comment (zmstone) [reply src/emqx_connection.erl:592]: Yes

File: etc/zones.conf [hunk 2]
```diff
@@ -60,8 +60,7 @@ zone.external.force_shutdown_policy = 10000|32MB
 ## Depth so big may lead to subscribing performance issues.
 ##
 ## Value: Number [0-65535]
-## Default 7
-zone.external.max_topic_levels = 7
+## zone.external.max_topic_levels = 7
```
Comment (zmstone) [inline etc/zones.conf:63]: Was the value 7 released?

**Review by zmstone — APPROVED** [review]

File: etc/zones.conf (hunk 2 above)
Comment (JimMoen) [reply etc/zones.conf:63]: not yet, changed after tag [e4.2.9](https://github.com/emqx/emqx/releases/tag/e4.2.9)
the commit: [e57c30a](https://github.com/emqx/emqx/commit/e57c30a0b9699807a8c14f781ef1f09e01e41b1d)

**Pull request merged**